### PR TITLE
Split common-header tests

### PR DESCRIPTION
**Pull request by alex-deaconu**
##  Description
Some refactoring
Adding helper functions
Restructure some tests

Enable users tests
User tests disabled in CH repo
Configured for parallel builds based on staging env

Less warnings
More specific selectors for elements that have duplicates
Subcompany selection is conditional based on whether already in a Subcompany

## Motivation and Context
Speed up test suites imported from CH

## How Has This Been Tested?
No actual code/functionality changes in this PR

Tests are passing

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
NA

### CODE REVIEW

**Comment by alex-deaconu** [comment]
@ezequielc please review. Thanks!

File: test/e2e/common-header/cases/company-subcompanies.js [hunk 1]
```diff
@@ -140,33 +106,15 @@
...
+          commonHeaderPage.createSubCompany(subSubCompanyName);
         });
         
         it("Switch to sub-sub-company", function() {
-          commonHeaderPage.getProfilePic().click();
-          homepage.getChangeSubcompanyButton().click();
-          
-          helper.wait(selectSubcompanyModalPage.getSelectSubcompanyModal(), "Select Subcompany Modal");
-          helper.waitDisappear(selectSubcompanyModalPage.getLoader(), "Load Companies");
+          browser.get(companyUrl);
```
Comment (ezequielc) [inline test/e2e/common-header/cases/company-subcompanies.js]: @alex-deaconu Curious, why do you need to go back to the main company here? Thanks

Comment (alex-deaconu) [reply test/e2e/common-header/cases/company-subcompanies.js]: @ezequielc I'm not really sure... it was already there so I left it. However it makes sense to remove. If you don't mind, I will do so as part of my next PR. Thanks.

**Comment by alex-deaconu** [comment]
@ezequielc remove the extra line of code however it was breaking the tests. I had to put in an additional check to support switching from a Subcompany to another. Thanks.

**Comment by ezequielc** [comment]
@alex-deaconu Sounds good. Thanks for fixing it.

**Pull request merged**